### PR TITLE
add RealT tokens

### PR DESCRIPTION
**Pull request by mastilver**
They only have the latest price using this API, I don't know if it's fine.
I set the confidence to 1, the price from the API is the price they are selling it to you for and they will buy it back from you with a 3% fee.

### CODE REVIEW

**Comment by waynebruce0x** [comment]
looks great. I'm tempted to set the price to the buy back price. My logic is that this is the price 'circulating' tokens are valued at. This is assuming that realT is the only available market. What do you think @mastilver ?

File: coins/src/adapters/other/index.ts [hunk 1]
```diff
@@ -73,3 +74,10 @@ export function manualInput(timestamp: number = 0) {
     manualInputAdapter("polygon", timestamp),
   ]);
 }
+export function realt(timestamp: number = 0) {
+  console.log("starting realt");
+  return Promise.all([
+    realtAdapter("ethereum", timestamp),
+    realtAdapter("gnosis", timestamp),
```
Comment (0xngmi) [inline coins/src/adapters/other/index.ts:81]: chain name might need to be xdai instead

Comment (mastilver) [reply coins/src/adapters/other/index.ts]: fixed :)

**Comment by Sigri44** [comment]
Look for our Community API : https://api.realt.community/

**Pull request merged**